src/ert/analysis/_es_update.py

```diff
@@ -22,7 +22,7 @@
 from ert.config import GenKwConfig
 
 from ..config.analysis_config import ObservationGroups, UpdateSettings
-from ..config.analysis_module import BaseSettings, ESSettings, IESSettings
+from ..config.analysis_module import BaseSettings, ESSettings
```

Comment:
Just merge the remaining two? Can always split it up again if we add something else that has overlap.

Reply:
Can also delete `IESSettings`